app/assets/javascripts/discourse/app/components/user-card-contents.js

```diff
@@ -56,9 +57,9 @@ export default Component.extend(CardContentsBase, CanCheckEmails, CleansUp, {
   },
 
   @discourseComputed("user.status")
-  userStatusEmoji() {
-    const emoji = this.user.status.emoji ?? "mega";
-    return emojiUnescape(`:${emoji}:`);
+  userStatusEmoji(status) {
```

Comment:
what if status doesn’t have an emoji? Also if we are certain about the emoji here, why not just compute on `user.status.emoji` ?

Reply:
We don't support status without emoji, so we are certain about emoji here. I changed it to making computation directly on `user.status.emoji`.